flink-runtime/src/main/java/org/apache/flink/runtime/leaderelection/ZooKeeperLeaderElectionService.java

```diff
@@ -397,8 +397,6 @@ protected void handleStateChange(ConnectionState newState) {
 				LOG.debug("Connected to ZooKeeper quorum. Leader election can start.");
 				break;
 			case SUSPENDED:
-				LOG.warn("Connection to ZooKeeper suspended. The contender " + leaderContender.getAddress()
```

Comment:
I don't think we should dismiss this log in this patch. Change it into

```
LOG.warn("Connection to ZooKeeper suspended. The contender " + leaderContender.getAddress()
                  + " is waiting for reconnection.");
```

might be ok.

Reply:
👍 Thanks, done